dev/com.ibm.ws.security.acme/resources/com/ibm/ws/security/acme/resources/AcmeMessages.nlsprops

```diff
@@ -10,19 +10,156 @@
+CWPKI2022E=CWPKI2022E: Failed to write to the domain key file for the ACME certificate authority account at URI {0}. The file location is {1}. The error is {2}.
+CWPKI2022E.explanation=Could not write to the domain key file for the certificate authority account.
+CWPKI2022E.useraction=Review the provided exception for details on the failure. Verify the file location is correct and has the correct file permissions.
+
+#"Error writing account key file."
+CWPKI2023E=CWPKI2023E: Failed to write the account key file for the ACME certificate authority account at URI {0}. The file location is {1}. The error is {2}.
+CWPKI2023E.explanation=Could not write to the account key file for the certificate authority account.
+CWPKI2023E.useraction=Review the provided exception for details on the failure. Verify the file location is correct and has the correct file permissions.
+
+# "Error revoking certificate."
+CWPKI2024E=CWPKI2024E: Failed to revoke the requested certificate for the ACME certificate authority at URI {0}. The certificate that failed to revoke has serial number {1}. The error is {2}.
+CWPKI2024E.explanation=A revoke request failed. The certificate may not be revoked.
+CWPKI2024E.useraction=Review the provided exception for details on the failure.
+
+#"Could not load account KeyPair. Revoking a certificate requires a valid account key pair."
+CWPKI2025W=CWPKI2025W: Received a certificate revoke request, but the account key pair for the ACME certificate authority at URI {0} could not be loaded.
+CWPKI2025W.explanation=Could not revoke a certificate because the account key pair could not be loaded.
+CWPKI2025W.useraction=Review the log for any earlier exceptions for details on the failure. Or manually review the key store file.
+
+# Unable to find account to revoke certificate.  Revoking a certificate requires a valid account.
+CWPKI2026W=CWPKI2026W: Received a certificate revoke request, but the account was not found at the ACME certificate authority at URI {0}.
+CWPKI2026W.explanation=Could not revoke a certificate because the account was not found.
+CWPKI2026W.useraction=Review the certificate authority URI provided. Or manually review the key store file.
 
-CWWKX0000I=CWWKX0000I: Test message
-CWWKX0000I.explanation= Test message description
-CWWKX0000I.useraction= No action is necessary at this time
\ No newline at end of file
+# "The " + type + " key file path must be valid."
+CWPKI2027E=CWPKI2027E: The ACME certificate authority {0} file path is invalid. The path provided is '{1}'.
+CWPKI2027E.explanation=The profile path was null or empty.
+CWPKI2027E.useraction=Provide a valid file path in the configuration.
+		
+# "Cannot read existing " + type + " key file."
+CWPKI2028E=CWPKI2028E: Failed to read the ACME certificate authority {0} file path. The path provided is '{1}'.
+CWPKI2028E.explanation=Could not read the provided file path.
+CWPKI2028E.useraction=Verify the file location is correct and has the correct file permissions.
```

Comment:
does this mean to verify that the file location has the correct file permissions?